### ISSUE TITLE
Emails are not sent with HTML formatting

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/jeffgeerling-com#92]
Unfortunately, it seems you are not able to send HTML email body content using Drupal 8 core like you could in Drupal 7. I installed the Simple Mail but ran into this issue: [Sent messages have HTML markup stripped, are plain text](https://www.drupal.org/project/simple_mail/issues/2777695).

**Comment by geerlingguy** [comment]
I tried just dropping the body text handling stuff Drupal was doing in a custom plugin (see above commit)... but that resulted in the mysterious error message:

```
Error sending email (from myemail@domain.com to myemail@domain.com with reply-to JeffGeerling.com <myemail@domain.com>).
``

So it's not as simple as swapping the MailInterface plugin.

**Comment by geerlingguy** [comment]
I'm also going to need to set up a local MailHog instance in Docker so I can more easily intercept and debug emails.

**Comment by geerlingguy** [comment]
I set up MailHog in the Docker environment in the above commit. Next up: Getting HTML email working.

**Comment by geerlingguy** [comment]
Fixed in #117! Will adjust if I notice anything go wrong.
